content/sensu-go/6.4/release-notes.md

```diff
@@ -85,6 +86,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.4.0 release notes
+
+**June 23, 2021** &mdash; The latest release of Sensu Go, version 6.4.0, is now available for download. 
+
+<!--// synopsis placeholder //-->
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.4.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][215]) In the web UI, the system information modal now includes license expiration information, accessed via the `CTRL + .` keyboard shortcut, for users with the appropriate permissions.
+- Added binary-only distribution for [macOS arm64][216].
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][215]) Add the ability to configure certain parameters per page in the Web UI.
```

Comment:
Mike, after learning a bit more about this item, I think it's better as a new feature. It adds a `page_parameters` object to the GlobalConfig resource for the web UI. I'm still collecting info to document it, but it is more substantial than I first thought. We should also add a bit to this description...pending the documentation.

Reply:
OK, I know a bit more now and will add a suggested revision...